README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
 =======================
 
 Core ML community tools contains all supporting tools for Core ML model
-conversion and validation. This includes scikit-learn, LIBSVM, Caffe,
-Keras and XGBoost.
+conversion, editing and validation. This includes scikit-learn, LIBSVM, Caffe,
+Keras, Tensorflow and XGBoost.
 
-coremltools 3.0
----------------
-[Release notes](https://github.com/apple/coremltools/releases/)
+# Table of contents
 
+* [Release information](#Release-Information)
+* [Neural network conversion guide](#Neural-network-conversion-guide)
+* [Core ML specification](#Core-ML-specification) 
+* [coremltools documentation](#coremltools-documentation-and-examples)
```

Comment:
Is this the user guide or documentation? Documentation in Python packages usually refers to API documentation. Might be better to call it "user guide and examples"

Reply:
Agreed

Reply:
Modified